input/data/2008/2008ApJ...679.1427A/tev-000014-sed.ecsv

```diff
@@ -0,0 +1,24 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_min, unit: TeV, datatype: float64}
```

Comment:
The corresponding author was a certain G.Maier and the values are from his notes... I've added a comment.